core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -107,6 +107,14 @@ class SparkHadoopUtil extends Logging {
         if (key.startsWith("spark.hadoop.")) {
           hadoopConf.set(key.substring("spark.hadoop.".length), value)
         }
+	// Copy any "fs.swift2d.foo=bar" properties into conf as "fs.swift2d.foo=bar"
```

Comment:
thats nice suggestion we add configs for azure also. I am not familiar with azure do you have a sample code that to understand and run and test